.vscode/launch.json

```diff
@@ -95,6 +107,31 @@
       "sourceMaps": true,
       "outFiles": ["${workspaceRoot}/packages/*/out/**/*.js"],
       "preLaunchTask": "Compile"
+    },
+    {
+      "name": "Launch Apex Debugger Tests",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceRoot}/packages/salesforcedx-vscode-apex-debugger/node_modules/mocha/bin/_mocha",
+      "args": [
+        "-u",
+        "tdd",
+        "--timeout",
+        "999999",
+        "--colors",
+        "--recursive",
+        "${workspaceRoot}/packages/salesforcedx-vscode-apex-debugger/out/test/"
+      ],
+      "sourceMaps": true,
+      "outFiles": ["${workspaceRoot}/packages/*/out/**/*.js"],
+      "preLaunchTask": "Compile",
+      "internalConsoleOptions": "openOnSessionStart"
+    }
+  ],
+  "compounds": [
+    {
+      "name": "Launch Extensions + Apex Debug adapter",
+      "configurations": ["Launch Extensions", "Launch Apex Debug adapter"]
```

Comment:
It will compile because it is simply running each of those configs. It will only compile once even though both configs specify a compile as a preLaunchTask.